doc/source/operations/control-plane-operation.rst

```diff
@@ -0,0 +1,393 @@
+
+Compute
+-------
+
+The compute nodes can largely be thought of as ephemeral, but you do need to
+make sure you have migrated any instances and disabled the hypervisor before
+rebooting, decommissioning or making any disruptive configuration change.
+
+Monitoring
+----------
+
+* `Back up InfluxDB <https://docs.influxdata.com/influxdb/v1.8/administration/backup_and_restore/>`__
+* `Back up OpenSearch <https://opensearch.org/docs/latest/tuning-your-cluster/availability-and-recovery/snapshots/snapshot-restore/>`__
+* `Back up Prometheus <https://prometheus.io/docs/prometheus/latest/querying/api/#snapshot>`__
+
+Seed
+----
+
+* `Back up bifrost <https://docs.openstack.org/kayobe/latest/administration/seed.html#database-backup-restore>`__
+
+Ansible control host
+--------------------
+
+* Back up service VMs such as the seed VM
+
+Control Plane Monitoring
+========================
+
+This section shows user guide of monitoring control plane. To see how to
+configure monitoring services, read :ref:`monitoring-service-configuration`.
+
+The control plane has been configured to collect logs centrally using Fluentd,
+OpenSearch and OpenSearch Dashboards.
+
+Telemetry monitoring of the control plane is performed by Prometheus. Metrics
+are collected by Prometheus exporters, which are either running on all hosts
+(e.g.  node exporter), on specific hosts (e.g. controllers for the memcached
+exporter or monitoring hosts for the OpenStack exporter). These exporters are
+scraped by the Prometheus server.
+
+Configuring Prometheus Alerts
+-----------------------------
+
+Alerts are defined in code and stored in Kayobe configuration. See ``*.rules``
+files in ``$KAYOBE_CONFIG_PATH/kolla/config/prometheus`` as a model to add
+custom rules.
+
+Silencing Prometheus Alerts
+---------------------------
+
+Sometimes alerts must be silenced because the root cause cannot be resolved
+right away, such as when hardware is faulty. For example, an unreachable
+hypervisor will produce several alerts:
+
+* ``InstanceDown`` from Node Exporter
+* ``OpenStackServiceDown`` from the OpenStack exporter, which reports status of
+  the ``nova-compute`` agent on the host
+* ``PrometheusTargetMissing`` from several Prometheus exporters
+
+Rather than silencing each alert one by one for a specific host, a silence can
+apply to multiple alerts using a reduced list of labels. Log into Alertmanager,
+click on the ``Silence`` button next to an alert and adjust the matcher list
+to keep only ``instance=<hostname>`` label.
+Then, create another silence to match ``hostname=<hostname>`` (this is
+required because, for the OpenStack exporter, the instance is the host running
+the monitoring service rather than the host being monitored).
+
+.. note::
+
+   After creating the silence, you may get redirected to a 404 page. This is a
+   `known issue <https://github.com/prometheus/alertmanager/issues/1377>`__
+   when running several Alertmanager instances behind HAProxy.
```

Comment:
I've never seen this before. Might be resolved?

Reply:
I agree